packages/custom_lint/lib/src/v2/custom_lint_analyzer_plugin.dart

```diff
@@ -396,6 +401,14 @@ class CustomLintServer {
     await clientChannel.init(
       debug: configs.any((e) => e != null && e.debug),
     );
+    _sendDelayedRequest();
+  }
+
+  void _sendDelayedRequest() {
+    for (final request in _delayedRequest) {
+      unawaited(_handleRequest(request));
```

Comment:
What's the point of all of this logic if we're unawaiting the requests anyway?

Reply:
This logic is same `_requestSubscription`, we just need replay request after delay, so no need to await or handle response. I also added test for this MR